scout/server/blueprints/variant/controllers.py

```diff
@@ -422,6 +422,52 @@ def variant_rank_scores(store, case_obj, variant_obj):
     return rank_score_results
 
 
+def get_loqusdb_obs_cases(store, variant_obj, category, obs_families=[]):
+    """Get a list of cases where variant observations occurred.
+    These are only the cases the user has access to.
+
+    Args:
+        store (scout.adapter.MongoAdapter)
+        variant_obj(scout.models.Variant) it's the variant the loqusdb stats are computer for
+        category(str)
+        obs_families(list). List of all cases in loqusdb where variant occurred
+
+    Returns:
+        obs_cases(list).
+    """
+    obs_cases = []
+    user_institutes_ids = set([inst["_id"] for inst in user_institutes(store, current_user)])
+    for i, case_id in enumerate(obs_families):
+        if len(obs_cases) > 10:
```

Comment:
Before this line was 
```
if i > 10:
break
```

But if you really want to collect 10 other cases with same variant observed this code was wrong, check at all the `continue` you have down the function. It's very likely that you will end up with less than 10 cases observed if you use i to break the loop! That's also the reason why you see more cases with observations when you use this branch.